src/runtime_src/core/pcie/linux/shim.h

```diff
@@ -181,8 +181,7 @@ class shim
      * Mapped CU register space for xclRegRead/Write(). We support at most
      * 128 CUs and each map is of 64k bytes.
```

Comment:
This comment still mentioned hard-coded 64k range. I think we can remove it now.